project_template/BUILDING.md

```diff
@@ -0,0 +1,12 @@
+
+## Requirements
+
+This azure sdk module has several external build-time requirements.
+
+- **Catch2** (for tests): https://github.com/catchorg/Catch2.git
+- **extra-cmake-modules**: https://github.com/KDE/extra-cmake-modules.git

```

Comment:
It seems undesirable to require any external dependencies to perform the core library build if possible. I'd like to see this removed as a core requirement and only possibly used when building "extra" stuff (such as documentation).